noodles-editor/src/noodles/components/op-components.tsx

```diff
@@ -1120,8 +1119,13 @@ function ContainerOpComponent({
   }
 
   const setCurrentContainerId = useNestingStore(state => state.setCurrentContainerId)
-  const nodes = useNodes()
```

Comment:
Odd regression considering this is a reactive hook (was looking at their [docs](https://reactflow.dev/api-reference/hooks/use-nodes), since other hooks of theirs aren't reactive.)

Are nested nodes in react-flow, but hidden? Or maybe it changes, but not when the container is mounted?